geonode/geoapps/forms.py

```diff
@@ -32,3 +32,20 @@ class Meta(ResourceBaseForm.Meta):
             'center_y',
             'data'
         )
+    
+    def __init__(self, *args, **kwargs):
```

Comment:
Is this necessary?

Reply:
It's required to correctly show the tooltips in metadata editing like:

![image](https://user-images.githubusercontent.com/51856725/120634038-1c48a600-c46b-11eb-8f23-8f7b34571502.png)